hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsRestOperation.java

```diff
@@ -25,6 +25,7 @@
 import java.net.UnknownHostException;
 import java.util.List;
 
+import org.apache.hadoop.classification.VisibleForTesting;
```

Comment:
that should go into the org.apache. import block. a lot of existing VisibleForTesting imports are in the "other" block, but that's from when we moved off guava by patching the source

Reply:
The import is already present in the class after merge with trunk, removed the redundant import.